README.md

```diff
@@ -33,50 +33,69 @@ demonstrate the functionalities of Scalar DB. The Q&A application is a web appli
 - Question Submission
 - Answering Submission
 
-Let's set up the environment for the demonstration so that you can try it out to have a better understanding of the functionalities. We will explain how to develop this application with Scalar DB and what differs to use Scalar DB with and without transaction.
+Let's set up the environment for the demonstration so that you can try it out to have a better understanding of the functionalities. 
+We will explain how to develop this application with Scalar DB and what differs to use Scalar DB with and without transaction.
+
+This document contains different underlying storage/database (such as cassandra, cosmos db) set up and configurations, Please follow any one of them.
 
 ## Set up the environment
 
-Please install [Scalar DB prerequisites](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started.md#install-prerequisites), [Node.js](https://nodejs.org/en/download/) and [yarn](https://yarnpkg.com/en/docs/install) before running this demonstration.
+Please install [Node.js](https://nodejs.org/en/download/)
 
-### Install the database model
+**For Cassandra**
+
+Please install [Scalar DB prerequisites](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-cassandra.md#install-prerequisites).
+
+OR 
 
-To install the database schema, we will use the [schema tools](https://github.com/scalar-labs/scalardb/tree/master/tools/schema) of ScalarDB.
-The schema files are located inside the [/database](https://github.com/scalar-labs/indetail/tree/master/Q%26A_application/database) folder.
+**For Cosmos DB**
 
-2 schema files are available, depending if you want to run the application using the [Storage](#use-scalar-db-without-transaction) or [Transaction](#use-scalar-db-with-transaction) mode.
+Please install [Scalar DB prerequisites](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-cosmosdb.md#install-prerequisites).
+
+Here we assume Oracle JDK 8, Node js and the underlying storage/database such as Cassandra are properly configured before starting this demonstration.
+
+### Install the database model
+    
+The schema files are located inside the [database/](database/) folder.
```

Comment:
```suggestion
The schema files are located in the [database](database) directory.
```

Reply:
Thank you for your suggestion.
Document modified accordingly.